nrf_802154_sl/include/fem/nrf_fem_control_config.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA. All Rights Reserved.
+ *
+ * The information contained herein is confidential property of Nordic Semiconductor ASA.
+ * The use, copying, transfer or disclosure of such information is prohibited except by
+ * express written agreement with Nordic Semiconductor ASA.
+ */
+
+#ifndef NRF_FEM_CONTROL_CONFIG_H_
+#define NRF_FEM_CONTROL_CONFIG_H_
+
+#include "nrf_fem_config.h"
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "drivers/nrfx_common.h"
+#include "drivers/nrfx_errors.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @section Configuration
+ */
+
+/**
+ * @brief Configures the PA and LNA device interface.
+ *
+ * This function sets device interface parameters for the PA/LNA module.
+ * The module can then be used to control a power amplifier or a low noise amplifier (or both) through the given interface and resources.
+ *
+ * The function also sets the PPI and GPIOTE channels to be configured for the PA/LNA interface.
+ *
+ * @param[in] p_config Pointer to the interface parameters for the PA/LNA device.
+ *
+ * @retval   ::NRF_SUCCESS                 PA/LNA control successfully configured.
```

Comment:
`NRFX_SUCCESS`? Same for other error codes in this header.

Reply:
Changed to NRFX_SUCCESS